.bazelrc

```diff
@@ -1,10 +1,5 @@
-# Bazel settings that apply to this repository.
-# Take care to document any settings that you expect users to apply.
-# Settings that apply only to CI are in .github/workflows/ci.bazelrc
-
-# Recommended when using copy_directory, see
-# https://github.com/aspect-build/bazel-lib/blob/main/docs/copy_directory.md
-startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
+# import common bazelrc shared with e2e workspaces
+import %workspace%/.bazelrc.common
```

Comment:
did you add a link to that .bazelrc.common in some docs? does it end up in users file somehow?

Reply:
broken out for use in e2e workspaces; wasn't targeting users with this one